docs/resources/logbook/2023_lhc.md

```diff
@@ -0,0 +1,28 @@
+# Timeline of LHC Run 3 Measurements in 2023
+
+This page provides quick access links to logbook entries of important OMC shifts or measurements done during the LHC 2023 commissioning and MDs.
+Some entries (underlined) offer a tooltip when hovered, with a quick description of the important information from that entry.
+The tables below can be sorted by clicking next to the column headers.
+
+!!! tip "Results Visualization"
+    Various results graphs and the scripts to make them are available in `lintrack`:
+    ```
+    /afs/cern.ch/eng/sl/lintrack/LHC_commissioning2023/
+    ```
+    
+| Date       |     Type      |         Shift Purpose          |                                                             Logbook Link                                                              |
+|:-----------|:-------------:|:------------------------------:|:-------------------------------------------------------------------------------------------------------------------------------------:|
+| 2023-03-28 | Commissioning |    Injection Linear Optics     |      [Shift Plan][inj_linear_optics]{target=\_blank .cern_login} /  [Summary][inj_linear_optics_sum]{target=\_blank .cern_login}      |
+| 2023-04-01 | Commissioning | Ramp and Squeeze Linear Optics | [Shift Plan][squeezed_linear_optics]{target=\_blank .cern_login} /  [Summary][squeezed_linear_optics_sum]{target=\_blank .cern_login} |                                                 |
+
+
+<!-- All the links below -->
+[inj_linear_optics]:                        https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=1081&dateFrom=2023-03-28T00%3A00%3A00&dateTo=2023-03-28T23%3A59%3A59&eventToHighlight=3739094 
+[inj_linear_optics_sum]:                    https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=1081&dateFrom=2023-03-29T00%3A00%3A00&dateTo=2023-03-29T23%3A59%3A59&eventToHighlight=3739274
+[squeezed_linear_optics]:                   https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=1081&dateFrom=2023-04-01T00%3A00%3A00&dateTo=2023-04-01T23%3A59%3A59&eventToHighlight=3741499 
+[squeezed_linear_optics_sum]:               https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=322&dateFrom=2023-04-01T00%3A00%3A00&dateTo=2023-04-01T23%3A59%3A59&eventToHighlight=3741722
+
+
+<!-- Tooltips -->
+*[Injection Linear Optics]: Injection optics with 2021 corrections validated. Kmod tests. DOROS-BPM & long AC-Dipole kicks.
```

Comment:
2022 corrections

Reply:
I don't think so:

LHCBEAM/ATS_2021_10_22_B1_GlobalBetaBeta_Inj | (double:1) -> 1.0
LHCBEAM/ATS_2021_10_22_B2_GlobalBetaBeta_Inj | (double:1) -> 1.0

Reply:
Hmm. Still I think these are 2021 beam tests corrections used in 2022 (for the injection ones)? As the restart after LS2 was 2022 I would tend to use that but I don't have a strong feeling about it.